src/java.base/share/classes/java/lang/invoke/LambdaFormEditor.java

```diff
@@ -648,57 +648,6 @@ LambdaForm collectArgumentsForm(int pos, MethodType collectorType) {
         return putInCache(key, form);
     }
 
-    LambdaForm collectArgumentArrayForm(int pos, MethodHandle arrayCollector) {
```

Comment:
It's counter-intuitive that removing a LFE tactic would be harmless.
Each LFE tactic is a point where LFs can be shared, reducing footprint.
But in this case `collectArgumentArrayForm` is always paired with
`collectArgumentsForm`, so the latter takes care of sharing.
The actual code which makes the arrays is also shared, via
`Makers.TYPED_COLLECTORS` (unchanged).